assignments/closure.js

```diff
@@ -1,9 +1,19 @@
 // ==== Challenge 1: Write your own closure ====
 // Write a simple closure of your own creation.  Keep it simple!
-
+function myName(){
+  let firstName = "Drew";
+  let lastName = "Johnson";
+  console.log(`Hello, my first name is ${firstName}.`);
+  function lastAndFirst(){
+    console.log(`My full name is ${firstName} ${lastName}.`);
+  }
+  lastAndFirst();
+}
+myName();
 
 // ==== Challenge 2: Create a counter function ====
 const counter = () => {
+  
   // Return a function that when invoked increments and returns a counter variable.
 };
```

Comment:
Do you understand the solution code on this one? It's an important concept. If not, let me know in a DM, and we can discuss it
```
const counter = () => {
  let count = 0;
  return () => (++count);
  // Return a function that when invoked increments and returns a counter variable.
};
```